pkg/operator/csidriveroperator/csioperatorclient/aws.go

```diff
@@ -8,15 +8,17 @@ import (
 )
 
 const (
-	AWSEBSCSIDriverName          = "ebs.csi.aws.com"
-	envAWSEBSDriverOperatorImage = "AWS_EBS_DRIVER_OPERATOR_IMAGE"
-	envAWSEBSDriverImage         = "AWS_EBS_DRIVER_IMAGE"
+	AWSEBSCSIDriverName              = "ebs.csi.aws.com"
+	envAWSEBSDriverOperatorImage     = "AWS_EBS_DRIVER_OPERATOR_IMAGE"
+	envAWSEBSDriverImage             = "AWS_EBS_DRIVER_IMAGE"
+	envAWSEBSDriverControlPlaneImage = "AWS_EBS_DRIVER_CONTROL_PLANE_IMAGE"
```

Comment:
Very tiny nit: Can you put this on a new line so as this constant stands out apart from others?